app/views/admin/taxonomies/_fields.html.erb

```diff
@@ -16,9 +16,9 @@
 
 <% if Directory.any? %>
     <div class="field">
-        <%= s.label :directories, class: "field__label" %>
+        <%= s.label :Destinations, class: "field__label" %>
         <legend class="field-group__legend">
-        <p class="field__hint">Choose which directories this taxonomy should appear in on Scout:</p>
+        <p class="field__hint">Choose which destinations this taxonomy should appear on</p>
```

Comment:
I'd like to keep the wording for directories as we have it for now, minus the reference to Scout - given we still aren't sure what the best terminology is, I think it's better to keep it all consistent until we've decided on whether to change it to something else. Otherwise we'll end up with directories, destinations, teams and everything else and it will get very confusing!

Reply:
gotcha - done!

Reply:
Thank you <3